src/tooling/piral-cli-webpack5/src/configs/pilet.ts

```diff
@@ -2,7 +2,7 @@ import * as webpack from 'webpack';
 import * as TerserPlugin from 'terser-webpack-plugin';
 import * as OptimizeCSSAssetsPlugin from 'optimize-css-assets-webpack-plugin';
 import { PiletSchemaVersion } from 'piral-cli';
-import { PiletWebpackPlugin } from 'pilet-webpack-plugin';
+import { piletWebpackConfigEnchancer } from 'pilet-webpack-config-enchancer';
```

Comment:
Do you mean `pilet-webpack-config-enhancer`?

Reply:
I mean the word `enchancer` is used consistently here, but I think its a typo and `enhancer` was meant.